src/components/10-atoms/progress-bar/pw.ui.test.js

```diff
@@ -0,0 +1,55 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-progress-bar';
+
+describe('Progress Bar', () => {
+  it('should render small', async () => {
+    await openProgressBar();
+
+    const fontSize = await page.$eval(
+      'p',
+      el => window.getComputedStyle(el).fontSize
+    );
+
+    expect(fontSize).toBe('16px');
```

Comment:
Is this a copy paste or do you intentionally check the font size?

Reply:
When it is `small` the font-size changes, that's why it check for font-size